csi/Cargo.toml

```diff
@@ -14,21 +14,25 @@ tonic-build = "0.1.0"
 prost-build = "0.6.0"
 
 [dependencies]
+async-trait = "0.1.36"
 bytes = "0.5"
 bytesize = "1.0.0"
 chrono = "0.4.9"
 clap = "2.33.0"
 env_logger = "0.7"
+failure = "0.1"
```

Comment:
@blaisedias added this one crate iirc but  after noticeing its deprecated changed to something else. So do we still need it?

Reply:
It's still used by our nvmeadm crate.